src/wallet/wallet_rpc_server.cpp

```diff
@@ -599,6 +606,13 @@ namespace tools
       uint64_t mixin = m_wallet->adjust_mixin(req.mixin);
       std::vector<wallet2::pending_tx> ptx_vector = m_wallet->create_transactions_2(dsts, mixin, req.unlock_time, req.priority, extra, req.account_index, req.subaddr_indices, m_trusted_daemon);
 
+      if (ptx_vector.empty())
+      {
+        er.code = WALLET_RPC_ERROR_CODE_TX_NOT_POSSIBLE;
+        er.message = "No transaction created";
+        return false;
+      }
+
```

Comment:
Doesn't the same check apply to `transfer_split`?

Reply:
The other ones seems fine. This one would have not crashed, but emitted an incorrect message. I think the others are fine, but I might have missed one.